DQMOffline/RecoB/python/dqmCollector_cff.py

```diff
@@ -3,18 +3,18 @@
 from DQMOffline.RecoB.dqmAnalyzer_cff import *
 
 #pf DATA collector
-bTagCollectorDATA = bTagHarvest.clone(ptRanges = cms.vdouble(0.0))
+bTagCollectorDATA = bTagHarvest.clone(ptRanges = (0.0,))
 # module execution
 bTagCollectorSequenceDATA = cms.Sequence(bTagCollectorDATA)
 
 #pf MC collector
 bTagCollectorMC = bTagHarvestMC.clone(
-    flavPlots = cms.string("allbcl"), #harvest all, b, c, dusg and ni histos 
-    ptRanges = cms.vdouble(0.0),
-    etaRanges = cms.vdouble(0.0),
+    flavPlots = "allbcl", #harvest all, b, c, dusg and ni histos 
+    ptRanges = (0.0,),
+    etaRanges = (0.0,),
```

Comment:
Is this syntax correct? Maybe yes... But why not sticking on the usual way of listing vectors in python with squared parentheses, e.g.:
```suggestion
    ptRanges = [0.0],
    etaRanges = [0.0],
```

Reply:
@perrotta Okay! I am considering changing parentheses as you mentioned. Thank you.